composer.json

```diff
@@ -4,7 +4,9 @@
     "description": "Neos plugin demonstrating a simple frontend login",
     "license": "MIT",
     "require": {
-        "neos/neos": "^5.0 || dev-master"
+        "neos/neos": "^5.0 || dev-master",
+        "neos/fusion-form": "^1.0 || dev-master"
```

Comment:
misses a comma

Reply:
Already adjusted sometime ago, you are probably on an old branch.

Reply:
🤦‍♂ I was looking at diffs here but only first commit...